linux_os/guide/system/software/sudo/sudo_custom_logfile/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Ensure Sudo Logfile Exists - sudo logfile'
+
+description: |-
+    A custom logfile can be configured for sudo with the logfile tag.
```

Comment:
Could you also add in the description what is expected to be configured and how?
Something like what is in the `ocil` is enough.

Having this in the description makes it easy to understand what this rule is about in the HTML guides and reports.